.github/workflows/release_ios.yml

```diff
@@ -0,0 +1,46 @@
+name: Release ios
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    name: release ${{ matrix.target }} for feature ${{ matrix.feature }}
+    env:
+      PROJECT_NAME_UNDERSCORE: abe_gpsw
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: aarch64-apple-darwin
+            feature: ffi
+          - target: x86_64-apple-darwin
+            feature: ffi
+          - target: aarch64-apple-ios
+            feature: ffi
+          # - target: armv7-apple-ios
+          #   feature: ffi
+          - target: x86_64-apple-ios
+            feature: ffi
+          # - target: i386-apple-ios
+          #   feature: ffi
```

Comment:
It is not a leftover right? To be sure, these targets are commented because they don't work for the moment right

Reply:
I keep them here to let people know that they are not working and because I was not sure if they were needed for future (honestly I am pretty sure that we can remove them)